jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibExtensionTest.java

```diff
@@ -212,6 +212,8 @@ public void testContainer() {
     assertThat(container.getArgs()).containsExactly("arg1", "arg2", "arg3").inOrder();
     assertThat(container.getPorts()).containsExactly("1000", "2000-2010", "3000").inOrder();
     assertThat(container.getFormat()).isSameInstanceAs(ImageFormat.OCI);
+    container.setFormat("Docker");
```

Comment:
Let's move this line up to line 202. We want all the setup code to be in one place.

Reply:
if we move the line to 202, the set will override the original `setFormat(ImageFormat.OCI)`, it will make the line 214 wrong, and not cover the code `setFormat(ImageFormat format)`, so I write the `setFormat("Docker")` here.

Reply:
and a little confuse: why kokro-macos always failed, first time I thought it is no test code, but after added test code , it still failed. Thanks

Reply:
You're right! Sorry about that. In that case, it would be much better to have this new test be in a test method of its own. It is better to have smaller and more focused tests. We can name it `testSetFormat()`.

Reply:
Unfortunately we're experiencing some failures in our macos job which we're currently trying to address. I don't think it has anything to do with your PR.

Reply:
> You're right! Sorry about that. In that case, it would be much better to have this new test be in a test method of its own. It is better to have smaller and more focused tests. We can name it `testSetFormat()`.

Got it, I will done it soon.